### PR TITLE
WIP: Create .travis.yml

### PR DESCRIPTION
**Pull request by EwoutH**
`Cargo.toml` is hardcoded, which makes testing on forks difficult. Could you enable [Travis CI](https://github.com/marketplace/travis-ci)?

### CODE REVIEW

**Comment by lu-zero** [comment]
in order to build the bindings you need to install dav1d first.

**Comment by EwoutH** [comment]
That's... very logical. I'm done for tonight, feel free to give it a try.

**Comment by lu-zero** [comment]
thank you for your hard work :) I'll try to come up with a script to download and install dav1d hopefully tomorrow.

**Comment by EwoutH** [comment]
Nope this is something for tomorrow.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+language: rust
+
+rust:
+  #- stable ## Let's spare Travis
+ - nightly
+
+before_install:
+ - wget https://code.videolan.org/videolan/dav1d/-/archive/0.1.0/dav1d-0.1.0.tar.gz
+ - tar -xvf dav1d-0.1.0.tar.gz
```
Comment (lu-zero) [inline .travis.yml]:
```suggestion
 - tar -xvf dav1d-0.1.0.tar.gz
 - cd dav1d-0.1.0
 - meson .build
 - ninja -C .build
 - sudo ninja -C .build install
```

maybe?

**Comment by lu-zero** [comment]
You are right! pip3 sounds the simplest way.

**Comment by lu-zero** [comment]
I'd merge the broken test and then edit in the fixes from the github ui now.

**Pull request merged**